### PR TITLE
fix: renterd file health slab keys

### PR DESCRIPTION
**Pull request by alexfreska**
- Fixed an issue where file health slab keys were showing up as repeated and incorrect.

### CODE REVIEW

**Comment by alexfreska** [comment]
* **#602** <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/602?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alexfreska and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by alexfreska** [comment]
@ChrisSchinnerl PR for https://github.com/SiaFoundation/web/issues/596. Just want to to confirm, object slab keys should be unique? As in object will not have 2 slabs with the same key?

File: apps/renterd/components/Files/Columns/FilesHealthColumn/FilesHealthColumnContents.tsx [hunk 1]
```diff
@@ -57,7 +57,6 @@ export function FilesHealthColumnContents({
   const slabs = sortBy(
     obj.data.object.slabs?.map((s) => ({
       ...s.slab,
-      key: `${s.offset}${s.length}${s.slab.key}`,
```
Comment (alexfreska) [inline apps/renterd/components/Files/Columns/FilesHealthColumn/FilesHealthColumnContents.tsx:60]: I think in the past there was a weird issue where some objects had slabs with the same key (so added this "React `key`" for uniqueness, but forgot the name conflicted with the actual `key`), I assume that keys are actually supposed to be unique across the object so removing this temp fix which also fixes the observed issue too.

**Comment by ChrisSchinnerl** [comment]
> @ChrisSchinnerl PR for #596. Just want to to confirm, object slab keys should be unique? As in object will not have 2 slabs with the same key?

It's not impossible but highly unlikely. e.g. an object uploaded with tiny multipart uploads might reference the same slab over and over but at different offsets and lengths.

**Comment by alexfreska** [comment]
> > @ChrisSchinnerl PR for #596. Just want to to confirm, object slab keys should be unique? As in object will not have 2 slabs with the same key?
> 
> It's not impossible but unlikely. e.g. an object uploaded with tiny multipart uploads might reference the same slab over and over but at different offsets and lengths. So we should not assume that they are always unique.

@ChrisSchinnerl ah good to know, I must have hit this issue. I will add back the unique `id` I had but name it `id`.

File: apps/renterd/components/Files/Columns/FilesHealthColumn/FilesHealthColumnContents.tsx (hunk 1 above)
Comment (alexfreska) [reply apps/renterd/components/Files/Columns/FilesHealthColumn/FilesHealthColumnContents.tsx:60]: Nevermind, Chris confirmed that not necessarily unique - adding back.

**Review by ChrisSchinnerl — APPROVED** [review]

**Pull request merged**